test/unit/adapters/http.js

```diff
@@ -191,6 +226,22 @@ describe('supports http with nodejs', function () {
     });
   });
 
+  it('should support brotli error handling', function (done) {
+    if(typeof zlib.brotliCompress === 'function'){
```

Comment:
similar here, I'd suggest removing the conditional from the test

Reply:
I don't know if this is actually running, since `brotliCompress` is probably undefined?